x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/categorization_job_creator.ts

```diff
@@ -126,9 +128,38 @@ export class CategorizationJobCreator extends JobCreator {
     this._validationChecks = validationChecks;
     this._overallValidStatus = overallValidStatus;
 
+    this._ccsVersionFailure = this._checkCcsFailure(examples, overallValidStatus, validationChecks);
+    if (this._ccsVersionFailure === true) {
+      // if the index pattern contains a cross-cluster search, one of the clusters may
+      // be on a version which doesn't support the fields API (e.g. 6.8)
+      // and so the categorization examples endpoint will fail
+      // if this is the case, we need to allow the user to progress in the wizard.
+      this._overallValidStatus = CATEGORY_EXAMPLES_VALIDATION_STATUS.VALID;
+    }
+
     this._wizardInitialized$.next(true);
 
-    return { examples, sampleSize, overallValidStatus, validationChecks };
+    return {
+      examples,
+      sampleSize,
+      overallValidStatus,
+      validationChecks,
+      ccsVersionFailure: this._ccsVersionFailure,
+    };
+  }
+
+  // Check to see if the examples failed due to a cross-cluster search being used
+  private _checkCcsFailure(
+    examples: CategoryFieldExample[],
+    status: CATEGORY_EXAMPLES_VALIDATION_STATUS,
+    checks: FieldExampleCheck[]
+  ) {
+    return (
+      examples.length === 0 &&
+      status === CATEGORY_EXAMPLES_VALIDATION_STATUS.INVALID &&
+      checks[0]?.id === VALIDATION_RESULT.NO_EXAMPLES &&
+      this.indexPatternTitle.includes(':')
```

Comment:
I know this check is trivial, but now we are using it to check for cross-cluster index patterns in two places (see `public/application/data_frame_analytics/pages/analytics_management/components/source_selection/source_selection.tsx`), is it worth adding to somewhere like `index_utils`?

Reply:
Changed in https://github.com/elastic/kibana/pull/97855/commits/646ecbff4ef10e9d8862bd5f9a196df83b12a36e